.github/workflows/regression-tests.yml

```
@@ -21,9 +21,14 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - name: Find pull request
-        uses: jwalton/gh-find-current-pr@v1.3.3
-        id: finder
+      - name: Pull Request Details
+        id: pr_details
+        uses: actions/github-script@v7
+        with:
+          retries: 3
+          script: |
+            const prData = require('./.github/workflows/prData.js');
```

hyphen + expand the name
```suggestion
            const prData = require('./.github/workflows/pull-request-data.js');
```